src/frontend/src/app/components/login/login.component.ts

```diff
@@ -12,15 +13,15 @@ export class LoginComponent implements OnInit {
   public errors: any = [];
 
   constructor(
-    private fb: FormBuilder,
+    private fb: FormBuilder, private _userService: UserService
     ){}
    
 
   ngOnInit() {
 
     this.loginForm = this.fb.group({
       userGroup: this.fb.group({
-        email: ['', [Validators.required, Validators.email, Validators.maxLength(200)]]
```

Comment:
Why are we removing the validation for email addresses?

Reply:
I thought if we remove validator user can use either of one username or email address to login into the website. 
If the only email address is used to log in is the mandatory scenario, then I will implement validation. I don't remember exactly from the meeting with the stem-echo system.